yoga/Yoga.cpp

```diff
@@ -3019,8 +3025,7 @@ static void YGNodelayoutImpl(const YGNodeRef node,
   }
 
   // STEP 8: MULTI-LINE CONTENT ALIGNMENT
-  if (performLayout && (lineCount > 1 || YGIsBaselineLayout(node)) &&
-      !YGFloatIsUndefined(availableInnerCrossDim)) {
+  if (performLayout && lineCount > 1 && !YGFloatIsUndefined(availableInnerCrossDim)) {
```

Comment:
no need to enter this block even on baseline (except for multi line). Because the baseline calculation and child's cross position adjustment is already done in step6 and step7.